src/main/java/com/conveyal/datatools/manager/models/FeedSourceSummary.java

```diff
@@ -62,6 +62,8 @@ public class FeedSourceSummary {
 
     public LatestValidationResult latestValidation;
 
+    public String url;
```

Comment:
How is `url` supposed to be populated? This PR does not seem to change what fields are queried/extracted from Mongo.